src/it/java/com/ihren/processor/util/KafkaUtils.java

```diff
@@ -22,7 +24,13 @@
 
 @UtilityClass
 public class KafkaUtils {
-    public<K, V> V getRecord(KafkaConsumer<K, V> consumer, String topic, Duration timeout) {
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+
+    static {
```

Comment:
Don't use static block.
This field shouldn't be static and configuration should be done in private constructor